### ISSUE TITLE
Pain on the lateral aspect of the foot

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10743]
**Preferred term label:**
Pain on the lateral aspect of the foot

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Fifth metatarsal fracture
Calcaneus fractures

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001153
- label: Pain on the lateral aspect of the foot
- parents: [Foot pain (HP:0025238)](https://hpo.jax.org/app/browse/term/HP:0025238)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the side of the foot nearest to the little (fifth) toe.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10743
- synonyms: Lateral foot pain(NONE)
